### ISSUE TITLE
Radio break down on G.O.S Dariyah

### ISSUE THREAD

**Issue by omar2205** [IDI-Systems/acre2#516]
**Arma 3 Version:** `1.82` (stable)
**CBA Version:** `3.6.1` (stable)
**ACRE2 Version:** `2.5.1` (stable)

**Mods:**
```
- CBA_A3
- acre2
- ace3
- G.O.S Dariyah
- CUP (Core + Terrain)
- RHS
```

**Description:**
- When playing on G.O.S Dariyah, Radio breaks down a lot. It would be good for a few minutes and it would break. No one can use the radio (even if they are next to each other) We mostly use the 152 radio. 

- I thought it would be the terrain loss so I disabled it on the server and still the problem happens.

**Steps to reproduce:**
- Play on the map (G.O.S Dariyah) for a few minutes (~30mins)
- After a while, the radio will break, range doesn't matter.

**Where did the issue occur?**
- Dedicated

**Placed Modules:**
- None

**RPT log file:**
- Extension log: [gist](https://gist.github.com/omar2205/6f328c3976bc28d62b3354220a60c2c8)
- Sorry I don't have RPT log file (I can't seem to figure how to save it from the server)

**Comment by TheMagnetar** [comment]
We need the RPT to have a quick look at the issues. Many things can cause the radios to break, loadout, mission specific configurations like initialisation scripts, server entering a phase where no data can be sent etc..

Understand that we cannot always reproduce in person the issues that have been found, specially if this happens after 30 minutes of gameplay.

Are you using TADST for server management? If so, there is a folder named logs in the TADST folder of the A3 server directory. Can you also attach the client RPT?

**Comment by omar2205** [comment]
Sorry about the lack of rpt, I run the server directly from bash, I figured how to export the logs now, but I have to wait for the next OP to get the logs.

In the meanwhile, is there any info about how to debug this issue?

**Comment by Cuel** [comment]
When you say "can't use radio" do you mean that they hear the squelch and can transmit or does nothing happen at all when pressing the radio key?

**Comment by Soldia1138** [comment]
Can you confirm that the signal strength (in dBm) is in a valid range (above -120dBm roughly), but the signal quality is 0%? 

Can you say since when this problem occurs? Latest Arma Update or earlier?

**Comment by omar2205** [comment]
Before the update,
the signal sometimes ranges above and below 120

sorry I didn't document the numbers,
I can try to do some testing and get back to you.

**Comment by Soldia1138** [comment]
This should be solved by #524
